ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionManagementPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { PageSection } from '@patternfly/react-core';
+import { Route, Switch } from 'react-router-dom';
+
+import { exceptionManagementPath } from 'routePaths';
+
+import PageNotFound from 'Components/PageNotFound';
+import PageTitle from 'Components/PageTitle';
+import ExceptionRequestsPage from './ExceptionRequestsPage';
+
+function ExceptionManagementPage() {
```

Comment:
What do y’all think about moving toward a convention for container sub-routing component?

Page seems ambiguous with the requests page and a request page.

As a starting point for discussion, and I expect can be improved on:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsRoute.tsx

By the way, because this example did not use `Switch` element and because React Router 6 has `Routes` element instead, I did not use **Switch**. However, I am not sure how well **Route** reads.

Here is another place to consider for intuitive naming:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/MainPage/Body.tsx#L73-L78

Reply:
@pedrottimark I noticed that `AdministrationEventsRoute` isn't referenced anywhere. Is that a bug?

Reply:
Good eyew. Because dynamic loading is only reference:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/MainPage/Body.tsx#L73-L78

Reply:
Ahh, I see. That actually raises a good point about cleaning up code. What if someone notices that something isn't referenced and decides to delete that bit of code? That might cause some issues unless the code is tested well. I guess that gives more of a reason to test dynamically generated content.

Reply:
Also, on the subject of `Page` vs. `Route`. Was there a discussion about this before? It seems we have a few that use `Route`, but a majority use `Page`. What's your reasoning for using the term `Route`, @pedrottimark? Personally, I'm more inclined to go with `Page` because it describes the component better. In terms of sub-routing, I think I have a PR coming out that does make use of sub-routing. Maybe we can discuss if that approach works or not in that PR. For now, I'll go ahead and merge this, but let's keep the discussion going

Reply:
Yeah, no worries. For administration events, there are 2 page elements, so I ran out of names :)
* AdministrationEventsPage
* AdministrationEventPage